thirdparty/openapi3/components/schemas/ConsentsPostRequest.yaml

```diff
@@ -15,12 +15,12 @@ properties:
     description: |
       The id of the ConsentRequest that was used to initiate the
       creation of this Consent.
-  credential:
-    $ref: ./SignedCredential.yaml
+  
   scopes:
     type: array
     items:
       $ref: ./Scope.yaml
 required:
   - consentId
   - scopes
+  - consentRequestId  # we need to obligatory store the reference to request process
```

Comment:
I guess that it must of been a error in the docs since I saw it missing from the request
![image](https://user-images.githubusercontent.com/5932442/120643426-c4467b00-c43b-11eb-8fa8-ddec21df3164.png)

Reply:
`consentRequestId` is mandatory in Grant Consent phase when DFSP ask PISP to deliver credential
https://github.com/mojaloop/pisp/blob/master/docs/linking/README.md#15-grant-consent and `credential` not